doc-content/jbpm-docs/src/main/asciidoc/BPMN2/con-timer.adoc

```diff
@@ -0,0 +1,67 @@
+[id='con-timer_{context}']
+= Timers
+
+You can use timers to trigger a logic after a certain period or to repeat specific actions at regular intervals. Timers wait for a predefined amount of time before triggering once or repeatedly.
+
+== Configuration of timers with delay and period
+
+You can set a timer with delay and a certain period. The delay specifies the waiting time after the node activation, and the period defines the time between the subsequent trigger activation. The period value `0` results in a one-shot timer. You can specify the delay and period expression in `[#d][#h][#m][#s][#[ms]]` form, indicating the number of days, hours, minutes, seconds, and milliseconds (default). For example, the expression `1h` indicates one hour waiting time before triggering the timer again.
+
+== Configuration of timers with ISO-8601 date format
+
+You can configure timers with a valid ISO-8601 date format that supports both one-shot timers and repeatable timers. You can define timers as date and time representation, time duration, or repeating intervals. For example:
+
+* Date `2020-12-24T20:00:00.000+02:00` signifies that timer is triggered exactly on Christmas at 8:00 p.m.
+* Duration `PT1S` signifies that timer is triggered once after one second.
+* Repeating intervals `R/PT1S` signifies that timer is triggered every second with any limit. Alternatively, `R5/PT1S` triggers the timer five times every second.
+
+== Configuration of timers with process variables
+
+You can also specify timers using process variables, consisting of the string representation of delay and period or ISO8601 date format. When you specify ``#{variable}`, the engine extracts the process variable and accesses the process variable as a timer expression. In a process, you can use timers using the following ways:
+
+* Add a timer event to a process flow. The process activation starts the timer and the timer is triggered (once or repeatedly), it activates the successor of the timer node. Subsequently, the outgoing connection of a timer with a positive period value is triggered multiple times. When a timer node is canceled, the associated timer is also canceled and no more triggers occur.
+
+* Associate timer as a boundary event with a subprocess or task.
+
+== Update timers within running process instance
+
+In some cases, the scheduled timer needs to be rescheduled to accommodate the new requirements, such as changing delay, period, or repeat limit. Updating a timer includes many low-level operations, therefore, {PRODUCT} provides the following command to perform the low-level operations related to updating a timer as an atomic operation. The following command ensures that all the operations are performed within the same transaction.
+
+`org.jbpm.process.instance.command.UpdateTimerCommand`
+
+NOTE: The boundary timer events and intermediate timer events are supported to update.
```

Comment:
Does this mean `only` these timers are supported to update? If so (and for example the start timer events are not supported), I would add `only` in the note.

Reply:
Done